build.gradle

```diff
@@ -1144,6 +1144,7 @@ project(':raft') {
     testCompile project(':clients').sourceSets.test.output
     testCompile libs.junit
     testCompile libs.mockitoCore
+    testCompile libs.easymock
```

Comment:
Could we just use mockito consistently? We are trying to move away from easymock in general.